src/panels/lovelace/common/generate-lovelace-config.ts

```diff
@@ -39,38 +40,42 @@ const DOMAINS_BADGES = [
 ];
 const HIDE_DOMAIN = new Set(["persistent_notification", "configurator"]);
 
-const computeCards = (title: string, states: HassEntity[]): CardConfig[] => {
+const computeCards = (
+  title: string,
+  states: Array<[string, HassEntity]>
+): CardConfig[] => {
   const cards: CardConfig[] = [];
 
   // For entity card
   const entities: string[] = [];
 
-  states.forEach((stateObj) => {
-    const domain = computeStateDomain(stateObj);
+  for (const [entityId /*, stateObj */] of states) {
```

Comment:
I am commenting out to show that this variable is available. We might need it in the future and I want people to not forget it .